README.md

```diff
@@ -1,8 +1,78 @@
-# rclnodejs
+# rclnodejs - ROS Client Library for JavaScript language
 
-ROS2.0 Client Library for JavaScript with nodejs
+Branch | Linux Build | MacOS Build | Windows Build |
+------------ | :-------------: |  :-------------: |  :-------------: |
+develop | [![Build Status](https://travis-ci.org/RobotWebTools/rclnodejs.svg?branch=develop)](https://github.com/RobotWebTools/rclnodejs/tree/develop) | N/A | N/A
+master | [![Build Status](https://travis-ci.org/RobotWebTools/rclnodejs.svg?branch=master)](https://github.com/RobotWebTools/rclnodejs/tree/master) | N/A | N/A
 
-Branch | Build Status |
------------- | :-------------: |
-develop | [![Build Status](https://travis-ci.org/RobotWebTools/rclnodejs.svg?branch=develop)](https://github.com/RobotWebTools/rclnodejs/tree/develop)
-master | [![Build Status](https://travis-ci.org/RobotWebTools/rclnodejs.svg?branch=master)](https://github.com/RobotWebTools/rclnodejs/tree/master)
+## Build Environment
+
+1.Building ROS 2.
+
+Currently, the ROS is a cross-platform system, which covers Linux, MacOS and Windows, and the `rclnodejs` module is developed on the [`master`](https://github.com/ros2/ros2/blob/master/ros2.repos) branch of ROS, so you have to build ROS from source at the present stage. Please select the platform you want to work on, then reference the instruction of building ROS.
+
+* [Linux](https://github.com/ros2/ros2/wiki/Linux-Development-Setup)
+* [MacOS](https://github.com/ros2/ros2/wiki/OSX-Development-Setup)
+* [Windows](https://github.com/ros2/ros2/wiki/Windows-Development-Setup)
+
+2.Install `Node.js`.
+
+The `Node.js` version we selected is the latest LTS `Boron`. You can install it:
+
+* Download from Node.js offical [website](https://nodejs.org/en/), and install it.
+* Use the Node Version Manager ([nvm](https://github.com/creationix/nvm)) to install it.
+
+3.Install `node-gyp`, the build tool for `rclnodejs`.
+
+```javascript
+npm install -g node-gyp
```

Comment:
Actually, from npm 2.15.12, `node-gyp` was brought to `Node.js` 4.x. See [this change](https://github.com/npm/npm/blob/771c66884a0a1851cffb3bf132828f6ccef1c09c/changelogs/CHANGELOG-2.md#v21512-2017-03-24).

Reply:
Wow, I've not noticed this, so `node-gyp` doesn't need be mentioned here.